x-pack/solutions/security/plugins/security_solution/public/detections/components/attacks/table/table_section.test.tsx

```diff
@@ -377,4 +377,55 @@ describe('<TableSection />', () => {
       });
     });
   });
+
+  describe('enforced groups', () => {
+    it('should include enforcedGroups in groupingSettings', () => {
+      expect(groupingSettings.enforcedGroups).toBeDefined();
+      expect(Array.isArray(groupingSettings.enforcedGroups)).toBe(true);
+    });
+
+    it('should enforce ALERT_ATTACK_IDS as the enforced group', () => {
+      expect(groupingSettings.enforcedGroups).toEqual([ALERT_ATTACK_IDS]);
+    });
+
+    it('should pass enforcedGroups setting to GroupedAlertsTable', async () => {
+      render(
+        <TestProviders>
+          <TableSection dataView={dataView} statusFilter={[]} pageFilters={[]} />
+        </TestProviders>
+      );
+
+      await waitFor(() => {
+        expect(GroupedAlertsTable).toHaveBeenCalled();
+        const [props] = (GroupedAlertsTable as unknown as jest.Mock).mock.calls[0];
+        expect(props.settings.enforcedGroups).toEqual([ALERT_ATTACK_IDS]);
+      });
+    });
```

Comment:
As for this one, it basically renders the 2 previous test useless as it checks everything, but itself is also rendered useless by the last test below no?

Reply:
Good point - will remove it!

Reply:
done in https://github.com/elastic/kibana/pull/246389/commits/855c235679b0e69397f0698b327f04ee85365617